pykafka/client.py

```diff
@@ -59,8 +57,8 @@ def __init__(self,
             milliseconds) when reading responses for offset commit and
             offset fetch requests.
         :type offsets_channel_socket_timeout_ms: int
-        :param ignore_rdkafka: Don't use rdkafka, even if installed.
-        :type ignore_rdkafka: bool
+        :param ignore_rdkafka: Deprecated, and in fact ignored. Instead, you
```

Comment:
Argh, yes, I really should have thought of making this removal a separate pullreq. It would have been in time for 2.0.0. Somehow I totally didn't think of that :|